glean-core/android/src/test/java/mozilla/telemetry/glean/TestUtil.kt

```diff
@@ -319,3 +319,40 @@ fun <T> capture(value: ArgumentCaptor<T>): T {
 
 @Suppress("UNCHECKED_CAST")
 internal fun <T> uninitialized(): T = null as T
+
+/**
+ * Waits for ping with the given name to be received
+ * in the test ping server.
+ *
+ * @param pingName the name of the ping to wait for
+ * @param maxAttempts the maximum number of attempts
+ */
+fun waitForPingContent(
+    pingName: String,
+    pingReason: String?,
+    server: MockWebServer,
+    maxAttempts: Int = 3
+): JSONObject?
+{
+    var parsedPayload: JSONObject? = null
+    var attempts = 0
+    do {
```

Comment:
Is this loop different from `for (attempts in 1..maxAttempts)`?

Reply:
No, good point.  I actually copied this from the [test utilities for the sample app...](https://github.com/mozilla/glean/blob/main/samples/android/app/src/androidTest/java/org/mozilla/samples/gleancore/pings/SharedTestUtils.kt#L65)